### PR TITLE
Mobile design

### PR DESCRIPTION
**Pull request by waqaskanju**
-correct mobile version
- manipulated previous reviews.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,93 +10,138 @@
...
+                        <li><svg width="18" height="20" viewBox="0 0 18 20" fill="none"
+                                xmlns="http://www.w3.org/2000/svg">
+                                <path
+                                    d="M17.9852 7.46114C17.9852 6.32124 17.6192 5.25043 16.9205 4.31779C17.1867 3.41969 17.2532 2.27979 17.1867 0.967185C17.1534 0.414508 16.7209 0 16.1885 0C15.8891 0 13.3604 0.0345423 11.8965 1.38169C10.6322 1.1399 9.30128 1.1399 8.00368 1.38169C6.573 0.0345423 4.04435 0 3.71163 0C3.17928 0 2.74675 0.414508 2.71348 0.967185C2.61366 2.27979 2.71348 3.41969 2.97965 4.31779C2.28095 5.28497 1.91496 6.35579 1.91496 7.46114C1.91496 9.8791 3.71163 12.0553 6.43991 13.0915C6.3401 13.2988 6.27355 13.5406 6.20701 13.7824C3.31237 13.4715 1.94823 10.7427 1.88169 10.639C1.64879 10.1209 1.04989 9.91364 0.550818 10.19C0.0517425 10.4318 -0.147888 11.0535 0.118286 11.5717C0.184829 11.7444 2.01477 15.4404 6.04065 15.8549V18.9637C6.04065 19.5509 6.47318 20 7.0388 20H12.8614C13.427 20 13.8595 19.5509 13.8595 18.9637V14.8532C13.8595 14.2314 13.7264 13.6442 13.4935 13.1261C16.1885 12.0553 17.9852 9.91364 17.9852 7.46114Z"
+                                    fill="#505F79" />
```
Comment (roshan-bajgain) [inline index.html:102]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). SO kindly use `a ` tag on each iscons

**Review by roshan-bajgain — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 

Cheers and Happy coding!👏👏👏

## Optional Suggestion

- [ ] This is the last review for you so, I am making some of the changes in the optional. Strongly implement it. The section should have the proper background image (decorations are always backgrounding images, not `img` tags). so kindly use the CSS for styling the whole background image.
- [ ] I would suggest you use the same pull request for the next reviewer don't open the next pull request unless you moved to the next project.

__Check the comment under the review__

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: index.html (hunk 1 above)
Comment (waqaskanju) [reply index.html:102]: - Thanks man. First I will add <a> tag to it. 

- Second The only` img `in the in this files is of social media icon. I am not able to understand this point "decorations are always backgrounding images, not `img` tags". There are no image tag for background they are only the social media one.

- Third. When ever i change my branch the pull request change automatically. So your are says i have not to change a branch?

**Comment by roshan-bajgain** [comment]
@waqaskanju, thank you for your question. You have created a new pull request and this is your 3rd review so I didn't see another reviewer comment so don't open a pull request for re-submission kindly use the same one by fixing the error.

- [ ] second changes Just download whole background image and don't use it in HTML file as , you have to use CSS for it like .section {background-image: URL(....)} like this. kindly implement it.

**Pull request merged**